server/machines/machines.repository.go

```diff
@@ -0,0 +1,60 @@
+package machines
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type MachinesRepository struct {
+	Db *sql.DB
+}
+
+func (mr *MachinesRepository) GetAll() ([]Machine, error) {
+	rows, err = r.DB.Query("SELECT id, name, cpu_count, total_disk_space FROM machines")
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+	var res []*Machine
+	for rows.Next() {
+		var m Machine
+		if err := rows.Scan(&m.Id, &m.Name, &m.CpuCount, &m.TotalDiskSpace); err != nil {
+			return nil, err
+		}
+		res = append(res, &m)
+	}
+	if res == nil {
+		res = make([]*Machine, 0)
+	}
+	return res, nil
+}
+
+func (mr *MachinesRepository) GetOne(id string) (Machine, error) {
```

Comment:
`id` should be `uint32`
```suggestion
func (mr *MachinesRepository) FindById(id uint32) (MachinesEntity, error) {
```